Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -12,149 +12,239 @@ public static class HtmlReport
     {
         public static void CreateAxeHtmlReport(this IWebDriver webDriver, string destination)
         {
-            AxeBuilder axeBuilder = new AxeBuilder(webDriver);
+            var axeBuilder = new AxeBuilder(webDriver);
             webDriver.CreateAxeHtmlReport(axeBuilder.Analyze(), destination);
         }
 
         public static void CreateAxeHtmlReport(this IWebDriver webDriver, IWebElement context, string destination)
         {
-            AxeBuilder axeBuilder = new AxeBuilder(webDriver);
+            var axeBuilder = new AxeBuilder(webDriver);
             context.CreateAxeHtmlReport(axeBuilder.Analyze(context), destination);
         }
         public static void CreateAxeHtmlReport(this ISearchContext context, AxeResult results, string destination)
         {
             // Get the unwrapped element if we are using a wrapped element
             context = context is IWrapsElement ? (context as IWrapsElement).WrappedElement : context;
-            
-            HashSet<string> selectors = new HashSet<string>();
-            int violationCount = GetCount(results.Violations, ref selectors);
-            int incompleteCount = GetCount(results.Incomplete, ref selectors);
-            int passCount = GetCount(results.Passes, ref selectors);
-            int inapplicableCount = GetCount(results.Inapplicable, ref selectors);
+
+            var selectors = new HashSet<string>();
+            var violationCount = GetCount(results.Violations, ref selectors);
+            var incompleteCount = GetCount(results.Incomplete, ref selectors);
+            var passCount = GetCount(results.Passes, ref selectors);
+            var inapplicableCount = GetCount(results.Inapplicable, ref selectors);
 
             var doc = new HtmlDocument();
 
-            var node = HtmlNode.CreateNode("<html lang=\"en\"><head><meta charset=\"utf-8\"><title>Accessibility Check</title><style></style></head><body></body></html>");
-            doc.DocumentNode.AppendChild(node);
-
-            HtmlCommentNode hcn = doc.CreateComment("<!DOCTYPE html>\r\n");
-            HtmlNode htmlNode = doc.DocumentNode.SelectSingleNode("/html");
-            doc.DocumentNode.InsertBefore(hcn, htmlNode);
-
-            StringBuilder content = new StringBuilder();
-            content.AppendLine(@".fullImage{");
-            content.AppendLine($"content: url('{GetDataImageString(context)};border: 1px solid black;margin-left:1em;");
```

Comment:
Would prefer to keep this (image) in the CSS so it can be reused when embedding cropped images in the report results.

Reply:
Addressed, moved the image back into the css